README.md

```diff
@@ -1 +1,133 @@
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- JavaScript
+
+This project built with the help of HTML and JavaScript.
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Web pack]**
+- **[Dynamic Html]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+**This project is part of a training series to highlight the need for proper documentation, best practices, and writing clean codes with html, css and Javascript.**
+### Prerequisites
+- Have an IDE installed on your computer to be able to view the code locally.
+- Have a good working knowledge of Git and Github.
+
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone git clone https://github.com/unorjikingsley/To-Do-List.git
+- To get the curent working tree, git fetch --all, then checkout to to-doList-structure branch.
+- Finally, display the content of the index.html by running a live server or by just opening the file using a web browser.
+
+### Install
+- IDE preferrably WebStorm
+
+### Usage
+- For any individual who wishes to practice basic HTML,CSS and Javascript
+- For practice of HTML5, CSS and Jvascript technologies.
+### Deployment
+
+You can deploy this project using:git pages
+
+## 👥 Authors <a name="authors"></a>
+👤 **berylcl**
+- GitHub: [@berylcl](https://github.com/unorjikingsley)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>.
+
+- [ ] **[Database]**
+- [ ] **[More functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues pages](https://github.com/unorjikingsley/To-Do-List/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project... give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/unorjikingsley/To-Do-List6/blob/main/LICENSE) licensed.

```

Comment:
- [ ] This link is not working perfectly. Kindly add a `MIT.md` file and link it here. Here is a [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) file recommended by microverse. `MIT.md` is a license file that assures you are the owner of a specific project. So it is professional to add every project you own.